test/pjrt/test_dynamic_plugin_tpu.py

```diff
@@ -0,0 +1,34 @@
+import concurrent.futures
+
+from absl.testing import absltest
+import torch_xla.core.xla_model as xm
+import torch_xla.distributed.xla_multiprocessing as xmp
+from torch_xla.experimental import plugins
+import torch_xla.runtime as xr
+from torch_xla._internal import tpu
+
+plugins.register_plugin('TPU', tpu.TpuPlugin())
```

Comment:
are you going to put this in our init file eventually?

Reply:
Yeah. I'm avoiding any changes to the default behavior while this is WIP.